src/nifgen/metadata/functions_addon.py

```diff
@@ -35,6 +35,12 @@
 
 # Attach the given parameter to the given enum from enums.py
 functions_enums = {
+    'ConfigureOutputMode':               { 'parameters': { 1: { 'enum': 'OutputMode',                    }, }, },
```

Comment:
This conflicts with my work on #485.
I suggest you merge https://github.com/ni/nimi-python/tree/fgen-additional-metadata and then create system tests based on that.

Reply:
FYI there's a PR for this now
https://github.com/ni/nimi-python/pull/547